pkg/graph/resolvers/model_plan_phase.go

```diff
@@ -115,7 +119,24 @@ func TrySendEmailForPhaseSuggestion(
 		logger.Error(err.Error(), zap.Error(err))
 	}
 
-	return err
+	if currentPhaseSuggestion == nil {
```

Comment:
Do you know what case we'd call this with a `nil` `currentPhaseSuggestion`?

Wondering if it might be safer to just exit early in this case rather than clearing the suggestion.

Reply:
This is actually not achievable due to an earlier escape condition. I added this mostly to prevent a linter warning.

Reply:
Makes sense -- maybe we can just replace 
```go
		modelPlan.PreviousSuggestedPhase = nil
```
with
```go
		return nil
```

if that works for you?

Reply:
Changes are in